### DISCUSSION TITLE
After fine-tunning it doesn't sound the same

### DISCUSSION THREAD

**Discussion by borishar** [erew123/alltalk_tts#165] [Q&A]
What am I doing wrong? After fine tunning, testing in the next tab, I click move the the finetune button and it says its moved. Then when I load the ft model using curl, it loads, but it doesn't sound the same even though i am using the same wav that was selected during the test.

**Comment by erew123** [comment]
Hi @borishar 

Both Finetune and the main AllTalk script use exactly the same model loader setup, both load the config.json and vocab.json that is stored along with the model, which are the same ones used when finetuning as when in main AllTalk, so it should perform the same in both finetune and main AllTalk. 

The only thing you can do differently in the main AllTalk is to set a different temperature and repetition penalty and those would have an effect on the reproduction. Im assuming you havnt changed those settings?

![image](https://github.com/erew123/alltalk_tts/assets/35898566/07809efa-f5a4-4ca2-b1fd-35495143594f)

I could suggest trying with DeepSpeed disabled as well, but otherwise, they should be the same.

Beyond those things to try, I'm really not sure,

Thanks

**Comment by erew123** [comment]
Actually, there is one other thought I have. The fine-tuning loader has no text filtering on it, but main AllTalk does. I'm not sure what language you are using. Is it possible some characters or punctuation is filtered? You can obviously check what was sent to the model by looking at the command line/terminal output.
